File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.java
class Solution {
    public int triangularSum(int[] nums) {
       return solve(nums,nums.length-1); 
    }
    public int solve(int[] nums,int j){
        if(j==0) return nums[0];
        for(int i=0;i<j;i++){
            nums[i]=(nums[i]+nums[i+1])%10;
        }
        return solve(nums,j-1);

    }
}

File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.py
class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        return self.solve(nums,len(nums)-1)

    def solve(self,nums:List[int],j:int)->int:
        if(j==0):
            return nums[0]
        
        for i in range(j):
            nums[i]=(nums[i]+nums[i+1])%10

        return self.solve(nums,j-1)
        